monkey/monkey_island/cc/resources/agent_signals.py

```diff
@@ -0,0 +1,38 @@
+import logging
+from http import HTTPStatus
+from json import JSONDecodeError
+
+from flask import request
+
+from monkey_island.cc.event_queue import IIslandEventQueue, IslandEventTopic
+from monkey_island.cc.models import AgentSignals as Signal
```

Comment:
Any reason to do `as Signal`? Unless there's a name conflict, I usually avoid this, otherwise you need to look up what `Signal` is down in the code.

Reply:
Yeah, this resource class has the same name

Reply:
Ah, gotcha.

Reply:
`Signals` and not `Signal`, though, maybe?